client/reader/full-post/index.jsx

```diff
@@ -453,7 +456,7 @@ FullPostContainer = React.createClass( {
 		// This is a trick to make the dialog animations happy. We have to initially render the dialog
 		// as hidden, then set it to visible to trigger the animation.
 		process.nextTick( function() {
-			this.setState( { isVisible: true } ); //eslint-disable-line react/no-did-mount-set-state
+			this.props.showReaderFullPost();
```

Comment:
@blowery I'm not even sure that's needed, I've tried it with the defer and without, seems all the same to me. What you think?